### PR TITLE
example: panda css

### PR DESCRIPTION
**Pull request by frederikvonsperling**
# Panda CSS example
This example contains a simple implementation of [Panda CSS](https://panda-css.com/). It contains styling for a link made in four different approaches supported by Panda CSS. Refer to the [documentation](https://panda-css.com/docs) for when to use - and how to combine - the different approaches to suit your needs.

## Link examples
-  [Atomic Recipe (CVA)](https://panda-css.com/docs/concepts/recipes#atomic-recipe-or-cva) Create multi-variant atomic styles with a type-safe runtime API.
- [Atomic Style](https://panda-css.com/docs/concepts/writing-styles#atomic-styles) Type-safe reusable atoms with small CSS footprint.
- [Config Recipe](https://panda-css.com/docs/concepts/recipes#config-recipe) Resuable recipes extracted and generated just-in-time.
- [Text Styles](https://panda-css.com/docs/theming/text-styles#defining-text-styles) Global text styles for consistent and legible typography.

## Preview
Preview the example live on [https://example-panda-css.vercel.app/](https://example-panda-css.vercel.app/)

## Motivation
Panda CSS is a CSS-in-JS framework, that support the new era of server components and the rise of the app-directory in Next.js. It is a performant, type-safe, developer friendly framework, which uses modern CSS features. It is heavily inspired by existing frameworks, such as Chakra UI, Vanilla Extract, Stitches and Tailwind.

More on [why you should choose Panda CSS](https://panda-css.com/docs/overview/why-panda)

### CODE REVIEW

File: examples/panda-css/package.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+  "name": "panda-css",
+  "version": "0.1.0",
```
Comment (leerob) [inline examples/panda-css/package.json:3]:
```suggestion
```

File: examples/panda-css/package.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "next": "latest",
+    "react": "18.2.0",
+    "react-dom": "18.2.0"
+  },
+  "devDependencies": {
+    "@pandacss/dev": "^0.13.1",
```
Comment (leerob) [inline examples/panda-css/package.json:19]: Could we update all of these to their latest versions?

**Review by leerob — COMMENTED** [review]
Could you remove the lockfile please?

File: examples/panda-css/package.json (hunk 2 above)
Comment (frederikvonsperling) [reply examples/panda-css/package.json]: all dependencies updated

**Comment by frederikvonsperling** [comment]
> Could you remove the lockfile please?

@leerob lockfile removed and pr updated

**Review by leerob — APPROVED** [review]

**Pull request merged**